app/service/app_svc.py

```diff
@@ -87,6 +88,14 @@ def __init__(self, application, config):
         try:
             self.log.debug('Starting operation: %s' % operation.name)
             planner = await self._get_planning_module(operation)
+            if not operation.phases_enabled:
+                # Replace operation adversary with new Adversary whose
```

Comment:
move this conditional to a new private function, with a descriptive name, then remove the comments

Reply:
Alright, look at it now. Im not sold on the function name so feel free to suggest another. 

`async def _adjust_adversary_phases(self, operation):
        """If an operation has phases disabled, replace operation
        adversary with new adversary whose phases are collapsed.
        Modified adversary is temporary and not stored, just used
        for the operation.
        """
        if not operation.phases_enabled:
            return Adversary(adversary_id=(operation.adversary.adversary_id + "_phases_disabled"),
                             name=(operation.adversary.name + " - with phases disabled"),
                             description=(operation.adversary.name + " with phases disabled"),
                             phases={1: [i for phase, ab in operation.adversary.phases.items() for i in ab]})
        else:
            return operation.adversary`